src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -48,6 +50,37 @@ const defaultProps = {
 };
 
 function ReportActionItemCreated(props) {
+    const {windowWidth, isSmallScreenWidth} = useWindowDimensions();
+
+    const IMAGE_OFFSET_X = windowWidth / 2;
+    const IMAGE_OFFSET_Y = 75;
+    const ANIMATION_BOOST = 1.3;
+
+    // Get data from phone rotation sensor and prep other variables for animation
+    const animatedSensor = useAnimatedSensor(SensorType.GYROSCOPE);
+    const moveXoffset = useSharedValue(0);
+    const moveYoffset = useSharedValue(0);
+    const backgroundImageOffsetX = useSharedValue(-IMAGE_OFFSET_X);
+
+    // Apply data to create style object
+    const animatedStyles = useAnimatedStyle(() => {
+        /*
+         * We use pitch and roll instead of x and y because Reanimated makes these consistent across iOS and Android by standardizing on the iOS convention.
+         * For a visualization of what these values mean: https://howthingsfly.si.edu/flight-dynamics/roll-pitch-and-yaw
+         * These values are in radians
+         */
+        const {x, y} = animatedSensor.sensor.value;
+        // The x vs y here seems wrong but is the way to make it feel right to the user
+        moveXoffset.value = Math.min(IMAGE_OFFSET_X, Math.max(-IMAGE_OFFSET_X, moveXoffset.value - y * ANIMATION_BOOST));
+        moveYoffset.value = Math.min(IMAGE_OFFSET_Y, Math.max(-IMAGE_OFFSET_Y, moveYoffset.value - x * ANIMATION_BOOST));
+        if (isSmallScreenWidth) {
```

Comment:
Let's flip this around and return `{}` early if `!isSmallScreenWidth`:

```js
const animatedStyles = useAnimatedStyle(() => {
    if (!isSmallScreenWidth) {
        return {};
    }
    const {x, y} = animatedSensor.sensor.value;
    ...
    ...
});
```

This conforms better to our `prefer-early-return` JS style and also helps prevent any unnecessary computation (even if it does take a negligible amount of time)